SeriesHW.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<p>I think here you put a little bit too much knowledge from P1 into it, in not testing you uint at all</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_PythonCpp/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>